finality-provider/service/fp_instance.go

```diff
@@ -395,6 +389,18 @@ func (fp *FinalityProviderInstance) tryFastSync(targetBlock *types.BlockInfo) (*
 	return fp.FastSync(startHeight, targetBlock.Height)
 }
 
+func (fp *FinalityProviderInstance) fastSyncStartHeight(lastFinalizedHeight uint64) (uint64, error) {
```

Comment:
`fastSyncStartHeight` is called after we have known that there's finalized blocks, indicating that activation is already started, so maybe we don't need to check activation height here?

Reply:
fastSyncStartHeight returns the height that fast sync should start right? 
I probably can remove the check inside the fastSync than